.github/workflows/build-and-deploy-notifications.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches:
+    - main
+    - deploy-resources
+  pull_request:
+    branches:
+    - main
+
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build & test
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '6.0.x'
```

Comment:
This line has changed in `main` - we're on .NET 7 now.

Reply:
changed